diff.py

```diff
@@ -1544,6 +1550,10 @@ class ArchSettings:
     instructions_with_address_immediates=MIPS_BRANCH_INSTRUCTIONS.union({"jal", "j"}),
 )
 
+MIPSEL_SETTINGS = copy.deepcopy(MIPS_SETTINGS)
+MIPSEL_SETTINGS.name="mipsel"
+MIPSEL_SETTINGS.big_endian = False
```

Comment:
since ArchSettings uses dataclasses you can do `MIPSEL_SETTINGS = replace(MIPS_SETTINGS, name="mipsel", big_endian=False)`